src/components/TextField/TextField.module.css

```
@@ -0,0 +1,91 @@
+.input-wrapper__field {
+  box-sizing: border-box;
+  border: none;
+  outline: none;
+  padding: var(--component-input-space-padding-y)
+    var(--component-input-space-padding-x)
+    var(--component-input-space-padding-y);
+  width: 100%;
+}
+
+.input-wrapper {
+  background-color: var(--component-input-color-background-default);
+  border-width: var(--component-input-border-width-default);
+  border-style: solid;
+  box-sizing: border-box;
+  display: flex;
+  align-items: center;
+}
+
+.input-wrapper--default {
+  border-color: var(--component-input-color-border-default);
+  margin: var(--component-input-border-width-default);
+}
+
+.input-wrapper--default:hover {
+  border-color: var(--component-input-color-border-hover);
+}
+
+.input-wrapper--default:focus-within {
+  border-width: var(--component-input-border-width-focus);
+  border-color: var(--component-input-color-border-focus);
+  outline: none;
+  margin: 0px;
```

nitpick;
Dont need units for 0 values :) 

```suggestion
  margin: 0;
```